tools/utility/dockerise/run.py

```diff
@@ -162,6 +154,17 @@ def _run(**kwargs):
         # Choose the image
         docker_args.append(image)
 
+        # If we are cleaning then delete all files and directories in the build volume
+        if kwargs["clean"]:
+            if (
+                pty.spawn(
+                    docker_args + ["/bin/bash", "-c", "find /home/{}/build -mindepth 1 -delete".format(defaults.user)]
+                )
+                != 0
+            ):
+                cprint("Failed to clean the build volume", "red", attrs=["bold"])
+                exit(1)
```

Comment:
Personally, I prefer the style of naming and capturing the exit code that we used in `webots.py`. I think it reads a little better
```suggestion
            exit_code = pty.spawn(
                docker_args + ["/bin/bash", "-c", "find /home/{}/build -mindepth 1 -delete".format(defaults.user)]
            )
            if exit_code != 0:
                cprint("Failed to clean the build volume", "red", attrs=["bold"])
                exit(exit_code)
```